index.html

```diff
@@ -4,26 +4,192 @@
     <!-- Set the page title -->
     <title>Assignment 02</title>
 
-    <!-- Set the charset we're using to UTF-8 -->
-    <meta charset="UTF-8" />
-
-    <!-- Set the screens default zoom / scale to 1 -->
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <!-- Set the charset we're using to UTF-8 -->
+        <meta charset="UTF-8" >
     
-    <!-- Specifies the document mode for Internet Explorer -->
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-
-    <!-- We are using a CSS Reset file to standardize browser default css -->
-    <link rel="stylesheet" href="reset.css" />
-
-    <!-- This is where all of our awesome CSS code goes -->
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <!--
+        <!-- Set the screens default zoom / scale to 1 -->
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+        <!-- We are using a CSS Reset file to standardize browser default css -->
+        <link rel="stylesheet" href="reset.css">
+    
+        <!-- This is where all of our awesome CSS code goes -->
+        <link rel="stylesheet" href="style.css">
+      </head>
+      <body>
+        <div class="page">
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+          <!-- Our Header Section -->
+          <header id="home">
+            <nav>
+                <ul class="header">
+                    <li><a href="#home"><img src="logo.png" alt="Zach Miller"></a></li>
+                    <li><br><a href="#home">Home</a></li>
+                    <li><br><a href="#projects">My Projects</a></li>
+                    <li><br><a href="#about">About Me</a></li>
+                    <li><br><a href="#contact">Contact Me</a></li>
```

Comment:
Ideally, you should avoid using `<br>` tags to add space, and instead use CSS properties like padding and margin. 

The reason why is screen readers will read out "LINE BREAK" when they encounter a `<br />`.